### PR TITLE
Qe

### PR DESCRIPTION
**Pull request by hmeiland**

### CODE REVIEW

**Comment by xpillons** [comment]
@hmeiland you have pushed 2 apps. qe and quantumespresso, I guess one is enough :-)

File: apps/quantumespresso/readme.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Prerequisites
+
+Cluster is built with the desired configuration for networking, storage, compute etc. You can see the tutorial or examples folder in this repo for how to set this up.
+
+Dependencies for binary version:
+
+* None
+
+NOTE: Update the path to the lammps installer storageendpoint, sasurl and license server in $azhpc_dir/apps/lammps/install_lammps.sh
```
Comment (kanchanm) [inline apps/quantumespresso/readme.md]: this statement needs to be updated for QE

File: apps/quantumespresso/readme.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+```
+azhpc-scp -r $azhpc_dir/apps/. hpcuser@headnode:/apps
+```
+
+> Alternatively you can checkout the azurehpc repository but you will need to update the paths according to where you put it.
+
+## Installation
+
+```
+azhpc-run -u hpcuser -n "headnode compute" /apps/lammps/install_qe.sh
```
Comment (kanchanm) [inline apps/quantumespresso/readme.md]: apps/quantumespresso

File: apps/quantumespresso/readme.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+## Running
+
+NOTE: Update the path to the qe benchmark storageendpoint, saskey and license server in $azhpc_dir/apps/qe/run_qe_intelmpi.sh
+
+Now, you can run as follows:
+
+```
+qsub -l select=2:ncpus=15:mpiprocs=15,place=scatter:excl /apps/qe/run_qe_intelmpi.sh
```
Comment (kanchanm) [inline apps/quantumespresso/readme.md]: assuming you will use /apps/quantumespresso this needs to be updated

File: apps/quantumespresso/readme.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Connect to the headnode
+
+```
+azhpc-connect -u hpcuser headnode
+```
+
+## Running
+
+NOTE: Update the path to the qe benchmark storageendpoint, saskey and license server in $azhpc_dir/apps/qe/run_qe_intelmpi.sh
```
Comment (kanchanm) [inline apps/quantumespresso/readme.md]: assuming you will use /apps/quantumespresso this needs to be updated

File: apps/quantumespresso/run_qe_intelmpi.sh [hunk 5]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+DATA_DIR=/data
+CORES=`cat $PBS_NODEFILE | wc -l`
+PPN=`cat $PBS_NODEFILE | uniq -c | head -1 | awk '{ print $1 }'`
+MPI_HOSTFILE=$PBS_NODEFILE
+
+export qe_dir=AUSURF112
```
Comment (kanchanm) [inline apps/quantumespresso/run_qe_intelmpi.sh]: should we pass these as parameters?

File: apps/qe/run_qe_intelmpi.sh [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+export qe_dir=AUSURF112
+export qe_case=ausurf.in
+echo "downloading case ${qe_case}..."
+
+git clone https://github.com/QEF/benchmarks.git
+cd benchmarks 
+cd ${qe_dir}
+
+source /opt/intel/impi/*/bin64/mpivars.sh
```
Comment (edwardsp) [inline apps/qe/run_qe_intelmpi.sh]: this is h16r specific

File: apps/quantumespresso/run_qe_intelmpi.sh [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+export qe_case=ausurf.in
+echo "downloading case ${qe_case}..."
+
+git clone https://github.com/QEF/benchmarks.git
+cd benchmarks 
+cd ${qe_dir}
+
+source /opt/intel/impi/*/bin64/mpivars.sh
```
Comment (edwardsp) [inline apps/quantumespresso/run_qe_intelmpi.sh]: again, h16r specific (probably only want I_MPI_FABRICS now)

**Pull request closed without merging**